cmd/kepval/main_test.go

```diff
@@ -70,14 +73,61 @@ func TestValidation(t *testing.T) {
 		t.Fatal("must find more than 0 keps")
 	}
 
-	// Overwrite the command line argument for the run() function
-	os.Args = []string{"", ""}
-	for _, file := range files {
-		t.Run(file, func(t *testing.T) {
-			os.Args[1] = file
-			var b bytes.Buffer
-			if exit := run(&b); exit != 0 {
-				t.Fatalf("exit code was %d and not 0. Output:\n%s", exit, b.String())
+	kepParser := &keps.Parser{}
+	prrParser := &prrs.Parser{}
+	prrsDir := filepath.Join("..", "..", prrsDir)
+
+	for _, filename := range files {
+		t.Run(filename, func(t *testing.T) {
+			kepFile, err := os.Open(filename)
+			if err != nil {
+				t.Fatalf("could not open file %s: %v\n", filename, err)
+			}
+			defer kepFile.Close()
+
+			kep := kepParser.Parse(kepFile)
+			if kep.Error != nil {
+				t.Errorf("%v has an error: %v", filename, kep.Error)
+			}
+
+			requiredPRRApproval := len(kep.Number) > 0 && kep.LatestMilestone >= "v1.21"
```

Comment:
Just a query: Is there a way for a KEP to target v1.21 without a KEP number?

Reply:
Currently kep-number is not required.
But I want to make it required eventually (but we would need to migrate all keps to the new template).